### PR TITLE
Add support for language parameter

### PR DESCRIPTION
**Pull request by hnnweb**
Add support for language parameter.

Messed up the other pull request ( #14 )...

### CODE REVIEW

File: src/component/vue-hcaptcha.vue [hunk 1]
```diff
@@ -4,9 +4,12 @@
   </div>
 </template>
 <script>
-const CaptchaScript = (cb) => {
+const CaptchaScript = (cb, lang) => {
+    if(typeof lang === 'undefined') { // If not set, set english as default
```
Comment (e271828-) [inline src/component/vue-hcaptcha.vue:8]: I don't think this is the behavior you want: setting hl will force a language override, turning off browser autodetection.

Thus, this logic will force all users into English if you don't specify hl.

The correct behavior would be to simply not add hl if not specified.

Comment (brdlyptrs) [reply src/component/vue-hcaptcha.vue:8]: @e271828- is correct, hl should only be added to the string if the language was specified.

**Comment by brdlyptrs** [comment]
Looks good, just a small comment.

File: src/component/vue-hcaptcha.vue [hunk 2]
```diff
@@ -4,9 +4,13 @@
   </div>
 </template>
 <script>
-const CaptchaScript = (cb) => {
+const CaptchaScript = (cb, choosenlang) => {
+    let lang = '';
+    if(typeof lang !== 'undefined') { // Set the selected language
```
Comment (brdlyptrs) [inline src/component/vue-hcaptcha.vue:9]: I think you want `choosenlang` here, but really the simplest method is to do the following:
```js
script.async = true;
script.src = "https://hcaptcha.com/1/api.js?render=explicit"
if (choosenlang) 
    script.src += `&hl=${choosenlang}`;
```

**Comment by levz0r** [comment]
Hi everybody. Can you please review this change and merge? I've implemented this in the same way, then found this PR, so it looks good IMO.
Thanks in advance!

**Review by mtratsiuk — APPROVED** [review]

**Comment by mtratsiuk** [comment]
Would be also nice to update Readme and maybe add a test case for a new prop

File: src/component/vue-hcaptcha.vue [hunk 3]
```diff
@@ -4,11 +4,15 @@
   </div>
 </template>
 <script>
-const CaptchaScript = (cb) => {
+const CaptchaScript = (cb, choosenlang) => {
```
Comment (e271828-) [inline src/component/vue-hcaptcha.vue:7]:
```suggestion
const CaptchaScript = (cb, chosenlang) => {
```

File: src/component/vue-hcaptcha.vue [hunk 4]
```diff
@@ -4,11 +4,15 @@
...
     let script = document.createElement("script");
-    script.src = "https://hcaptcha.com/1/api.js?render=explicit";
     script.async = true;
-    
+    script.src = "https://hcaptcha.com/1/api.js?render=explicit";
+
+    if(typeof choosenlang !== 'undefined') {
+      script.src += `&hl=${choosenlang}`;
```
Comment (e271828-) [inline src/component/vue-hcaptcha.vue:13]:
```suggestion
    if(typeof chosenlang !== 'undefined') {
      script.src += `&hl=${chosenlang}`;
```

**Review by e271828- — APPROVED** [review]

**Pull request merged**